runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/GrpcFnServer.java

```diff
@@ -46,6 +46,11 @@
     return new GrpcFnServer<>(factory.create(service, endpoint), service, endpoint);
   }
 
+  public static <ServiceT extends FnService> GrpcFnServer<ServiceT> create(
```

Comment:
Mark this method as `@Deprecated` and clearly comment that this is for migration purposes for Dataflow.

Reply:
Fixed in commit https://github.com/apache/beam/pull/7168/commits/382e173a2d8aab6f045b3a213d3703b7104252b1